rules/five-user-experiences-of-reporting-services/rule.md

```diff
@@ -0,0 +1,44 @@
+---
+type: rule
+archivedreason: Only showing example screenshots
+title: Do you know the 5 user experiences of Reporting Services?
+guid: b66277bc-0091-402c-8d9b-ee3907eae909
+uri: five-user-experiences-of-reporting-services
+created: 2023-12-11T11:38:33.0000000Z
+authors: 
+  - title: Jeoffrey Fischer
+    url: https://ssw.com.au/people/jeoffrey-fischer
+related:
+- customization-do-you-know-which-version-of-sql-reporting-services-and-visual-studio-you-are-using
+redirects: []
+
+---
+
+<!--endintro-->
+
+The 5 user experiences of Reporting services are...
+
+* Vanilla
+
+<img src="VanillaExample.jpg" alt="Vanilla example" /><br/>
+<strong>Figure: Example of Vanilla user experience</strong>
+
+* Website
+
+<img src="WebsiteUExample.jpg" alt="Vanilla example" /><br/>
+<strong>Figure: Example of Website user experience</strong>
+
+* Email
+
+<img src="WebsiteUExample.jpg" alt="Vanilla example" /><br/>
+<strong>Figure: Example of Email user experience</strong>
+
+* Windows
+
+<img src="WebsiteUExample.jpg" alt="Vanilla example" /><br/>
+<strong>Figure: Example of Windows user experience</strong>
+
+* SharePoint
+
+<img src="WebsiteUExample.jpg" alt="Vanilla example" /><br/>
+<strong>Figure: Example of SharePoint user experience</strong>
```

Comment:
All the images seem old, is this intentional?

Reply:
Yes, we are migrating this and most of the images will be kept as is. However, some of them will be updated in upcoming rules.